### PR TITLE
[fix] Use KeyspaceName for CqlGenerator

### PR DESCRIPTION
**Pull request by sumtiogo**
whithout this patch, the following code will failed due to `No keyspace
has been specified.`

```csharp
using Cassandra;
using Cassandra.Mapping;
namespace Test
{
    class Program
    {
        static void Main(string[] args)
        {
            Cluster cluster = Cluster.Builder()
                .AddContactPoint("localhost")
                .Build();
            Mapper mapper = new Mapper(cluster.Connect());
            MappingConfiguration.Global.Define<MyMapping>();
            User u = mapper.First<User>("WHERE id = ?", 1);
        }
        public class MyMapping : Mappings
        {
            public MyMapping()
            {
                For<User>()
                    .KeyspaceName("test")
                    .TableName("user")
                    .Column(u => u.id, cm => cm.WithName("id"));
            }
        }
        public class User
        {
            public int id { get; set; }
        }
    }
}
```

### CODE REVIEW

**Comment by jorgebay** [comment]
Good catch! It looks like the issue affects all CQL generation using the Mapper.
Would you like to fix them all?
- Line 145
- Line 195
- Line 204
- Line 226

We could create a static method to get the table name and call it multiple times, similar to this:
```csharp
private static string GetEscapedTableName(PocoData pocoData)
{
    string name = null;
    if (Table.KeyspaceName != null)
    {
        name = Escape(Table.KeyspaceName, pocoData) + ".";
    }
    name += Escape(Table.Name, pocoData);
    return name;
}
```

Also, do you mind changing the commit message (avoid including a code snippet) and rebase?

File: src/Cassandra/Mapping/Statements/CqlGenerator.cs [hunk 1]
```diff
@@ -45,10 +45,11 @@ public void AddSelect<T>(Cql cql)
             var pocoData = _pocoDataFactory.GetPocoData<T>();
             var allColumns = pocoData.Columns.Select(Escape(pocoData)).ToCommaDelimitedString();
 
+            var identifier = string.IsNullOrEmpty(pocoData.KeyspaceName) ? pocoData.TableName : string.Format("{0}.{1}", pocoData.KeyspaceName, pocoData.TableName);
```
Comment (jorgebay) [inline src/Cassandra/Mapping/Statements/CqlGenerator.cs]: The fully-qualified escaped table name should be of the form: `"Ks1"."MyTable"` (not `"Ks1.MyTable"`)

**Comment by landim** [comment]
Looks good, whould you mind rebase to squash the commits into one*? 
* http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

**Comment by sumtiogo** [comment]
Done. Please check if that is what you want. Thanks.

**Comment by jorgebay** [comment]
Merged, thanks @sumtiogo !

**Comment by sumtiogo** [comment]
You're welcome. Thanks for developing this package, it helps me a lot.

**Pull request merged**